### DISCUSSION TITLE
Different behaviour in API rules between single and multiple relation

### DISCUSSION THREAD

**Discussion by lsalim19** [pocketbase/pocketbase#4211] [Q&A] [answered]
Hi,

I have an 'org' table that has an 'admin' field which is a multiple relation. If i set '@request.auth.id = admin' view rule than the record is not accessible but if I change the 'admin' field to a single relation everything works. I tried to change the rule to '@request.auth.id ?= admin' with no success. Thanks.

**Comment by ganigeorgiev** [comment, marked as answer]
When you use just the `relation` field it compares `@request.auth.id` with the raw value of the field.
For single `relation`, the raw value is a plain string with the id of related record (ex. `"id1"`).
For multiple `relation` the raw value is a serialized json array with the id of the related record(s) (ex. `["id1", "id2"]`).

In your case to make it work, you can try using dot-notation to target specifically the `id` of the related records, for example:
```js
@request.auth.id ?= admins.id
```

**Reply by lsalim19** [reply]
Thank you, this solution works perfectly!
